<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetailPage</class>
 <widget class="QWidget" name="DetailPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1370</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#DetailPage {
background: #FFFFFF;
border-radius: 20px;
}
</string>
  </property>
  <widget class="QTableWidget" name="tableWidget_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>330</width>
     <height>260</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget {
    alternate-background-color: #f2f7fb;

    background: #FFFFFF;

    font-family: PingFang SC;
    border: 0px;

    border-top-left-radius: 0px;
    border-top-right-radius: 0px;
    border-bottom-left-radius: 10px;
    border-bottom-right-radius: 10px;
}</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_1" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>0</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>0</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>0</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="widget_5" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="widget_6" native="true">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>290</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="widget_7" native="true">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>290</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>330</width>
     <height>260</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget {
    alternate-background-color: #f2f7fb;

    background: #FFFFFF;

    font-family: PingFang SC;
    border: 0px;

    border-top-left-radius: 0px;
    border-top-right-radius: 0px;
    border-bottom-left-radius: 10px;
    border-bottom-right-radius: 10px;
}</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_3">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>30</y>
     <width>330</width>
     <height>260</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget {
    alternate-background-color: #f2f7fb;

    background: #FFFFFF;

    font-family: PingFang SC;
    border: 0px;

    border-top-left-radius: 0px;
    border-top-right-radius: 0px;
    border-bottom-left-radius: 10px;
    border-bottom-right-radius: 10px;
}</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_4">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>30</y>
     <width>330</width>
     <height>260</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget {
    alternate-background-color: #f2f7fb;

    background: #FFFFFF;

    font-family: PingFang SC;
    border: 0px;

    border-top-left-radius: 0px;
    border-top-right-radius: 0px;
    border-bottom-left-radius: 10px;
    border-bottom-right-radius: 10px;
}</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>330</width>
     <height>432</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget {
    alternate-background-color: #f2f7fb;

    background: #FFFFFF;

    font-family: PingFang SC;
    border: 0px;

    border-top-left-radius: 0px;
    border-top-right-radius: 0px;
    border-bottom-left-radius: 10px;
    border-bottom-right-radius: 10px;
}</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_6">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>320</y>
     <width>330</width>
     <height>432</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget {
    alternate-background-color: #f2f7fb;

    background: #FFFFFF;

    font-family: PingFang SC;
    border: 0px;

    border-top-left-radius: 0px;
    border-top-right-radius: 0px;
    border-bottom-left-radius: 10px;
    border-bottom-right-radius: 10px;
}</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_7">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>320</y>
     <width>330</width>
     <height>432</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget {
    alternate-background-color: #f2f7fb;

    background: #FFFFFF;

    font-family: PingFang SC;
    border: 0px;

    border-top-left-radius: 0px;
    border-top-right-radius: 0px;
    border-bottom-left-radius: 10px;
    border-bottom-right-radius: 10px;
}</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
